### PR TITLE
BZs CVO-1

### PR DESCRIPTION
**Pull request by jeana-redhat**
BZs for CVO, batch 1

preview: https://deploy-preview-36991--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-9-release-notes.html#ocp-4-9-bug-fixes

### CODE REVIEW

File: release_notes/ocp-4-9-release-notes.adoc [hunk 1]
```diff
@@ -869,6 +869,15 @@ The deprecated `dhclient` binary has been removed from {op-system}. Starting wit
 
 *Cluster Version Operator*
 
+* Previously, the Cluster Version Operator (CVO) did not respect the `noProxy` property in the proxy configuration resource. As a result, the CVO was denied access to update recommendations or release signatures when only unproxied connections completed. Now, when the proxy resource requests direct, unproxied access, the CVO reaches the upstream update service and signature stores directly.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1978749[BZ#1978749])
```
Comment (stevsmit) [inline release_notes/ocp-4-9-release-notes.adoc:873]: BZ in brackets need to be in asterisks.

File: release_notes/ocp-4-9-release-notes.adoc [hunk 2]
```diff
@@ -869,6 +869,15 @@ The deprecated `dhclient` binary has been removed from {op-system}. Starting wit
...
+
+* Previously, the Cluster Version Operator (CVO) loaded its proxy configuration from proxy resource specification properties instead of from status properties that were verified by the Network Operator. As a result, any incorrectly configured values would prevent the CVO from reaching the upstream update service or signature stores. Now, the CVO loads its proxy configuration only from the verified status properties.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1978774[BZ#1978774])
```
Comment (stevsmit) [inline release_notes/ocp-4-9-release-notes.adoc:876]: BZ in brackets need to be in asterisks.

File: release_notes/ocp-4-9-release-notes.adoc [hunk 3]
```diff
@@ -869,6 +869,15 @@ The deprecated `dhclient` binary has been removed from {op-system}. Starting wit
...
+
+* Previously, the Cluster Version Operator (CVO) did not remove volume mounts that were added outside of the manifest. As a result, pod creation could fail during a volume failure. Now, CVO removes all volume mounts that do not appear in the manifest.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2004568[BZ#2004568])
```
Comment (stevsmit) [inline release_notes/ocp-4-9-release-notes.adoc:879]: BZ in brackets need to be in asterisks.

**Comment by stevsmit** [comment]
A few nits.

**Comment by jeana-redhat** [comment]
For https://issues.redhat.com/browse/OSDOCS-2514

**Pull request merged**